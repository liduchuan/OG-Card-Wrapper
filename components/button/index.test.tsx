import { render } from "@testing-library/react";
import { expect, test } from "vitest";
import { Button } from ".";

test("Button renders correctly", () => {
  const { container } = render(<Button />);
  expect(container).toMatchSnapshot();
});
