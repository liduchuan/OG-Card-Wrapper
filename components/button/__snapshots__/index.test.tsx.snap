// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Button renders correctly 1`] = `
<div>
  <button
    class="bg-amber-300"
  >
    hello world
  </button>
</div>
`;
